File: areas.py
nums = input()
numsArray = nums.split(' ')
numA = float(numsArray[0])
numB = float(numsArray[1])
numC = float(numsArray[2])

triangulo = (numA*numC)/2
circulo = 3.14159*numC*numC
trapezio = ((numA + numB)*numC)/2
quadrado = numB*numB
retangulo = numA*numB

print("TRIANGULO: {0:.3f}".format(triangulo))
print("CIRCULO: {0:.3f}".format(circulo))
print("TRAPEZIO: {0:.3f}".format(trapezio))
print("QUADRADO: {0:.3f}".format(quadrado))
print("RETANGULO: {0:.3f}".format(retangulo))

File: areas.c
#include <stdio.h>

int main(){
    float num1, num2, num3;
    scanf("%f%f%f", &num1,&num2,&num3);

    
    float triangulo = (num1*num3)/2;
    float circulo = 3.14159*num3*num3;
    float trapezio = ((num1 + num2)*num3)/2;
    float quadrado = num2*num2;
    float retangulo = num1*num2;

    printf("TRIANGULO: %.3f\n", triangulo);
    printf("CIRCULO: %.3f\n", circulo);
    printf("TRAPEZIO: %.3f\n", trapezio);
    printf("QUADRADO: %.3f\n", quadrado);
    printf("RETANGULO: %.3f\n", retangulo);
    return 0;
}